### DISCUSSION TITLE
How to set "ApplicableTradeTax.LineTotalBasisAmount"?

### DISCUSSION THREAD

**Discussion by jr-hoh** [stephanstapel/ZUGFeRD-csharp#443] [Q&A]
Hi everybody,

I have an invoice with allowances. I am setting `allowanceChargeBasisAmount` via `InvoiceDescriptor.AddApplicableTradeTax`. Valitool is reporting the following error:
[VD-Valitool-138]-Rechenfehler: ApplicableTradeTax -> BasisAmount = LineTotalBasisAmount + AllowanceChargeBasisAmount

Looking at the XML, `LineTotalBasisAmount` is missing. I found no way of setting this value. Is this simply not implemented or am I missing something?

**Comment by DaveHalma** [comment]
Hi CodingOtter, 
I did a 'workaround' by calculating it myself for each line. AFAIK, this is currently not implemented.
Dave
